tools/sklearn/lightgbm.xml

```diff
@@ -0,0 +1,291 @@
+            <param name="header2" value="True"/>
+            <param name="col2" value="1"/>
+            <param name="selected_task" value="train"/>
+            <param name="selected_algorithm" value="LightGBMRegressor"/>
+            <param name="objective" value="l2"/>
+            <param name="n_estimators" value="10000"/>
+            <param name="learning_rate" value="0.02"/>
+            <param name="max_leaf_nodes" value="32"/>
+            <param name="feature_fraction" value="0.9"/>
+            <param name="subsample" value="0.9"/>
+            <param name="max_depth" value="8"/>
+            <param name="lambda_l1" value="0.04"/>
+            <param name="lambda_l2" value="0.07"/>
+            <param name="min_gain_to_split" value="0.02"/>
+            <param name="min_child_weight" value="39.0"/>
+            <param name="verbose" value="-1"/>
+            <param name="random_state" value="1"/>
+            <output name="outfile_fit" file="lgb_regr_model.txt" compare="sim_size" delta="5"/>
+        </test>
+        <test>
+            <param name="infile_model" value="lgb_regr_model.txt" ftype="txt"/>
+            <param name="infile_data" value="regression_X.tabular" ftype="tabular"/>
+            <param name="selected_task" value="load"/>
+            <param name="header" value="True"/>
+            <output name="outfile_predict" file="lgb_prediction_result01.tabular"/>
+        </test>
+        <test>
+            <param name="infile1" value="train.tabular" ftype="tabular"/>
+            <param name="infile2" value="train.tabular" ftype="tabular"/>
+            <param name="col1" value="1,2,3,4"/>
+            <param name="col2" value="5"/>
+            <param name="selected_task" value="train"/>
+            <param name="selected_algorithm" value="LightGBMClassifier"/>
+            <param name="objective" value="binary"/>
+            <param name="n_estimators" value="10000"/>
+            <param name="learning_rate" value="0.02"/>
+            <param name="max_leaf_nodes" value="32"/>
+            <param name="feature_fraction" value="0.9"/>
+            <param name="subsample" value="0.9"/>
+            <param name="max_depth" value="8"/>
+            <param name="lambda_l1" value="0.04"/>
+            <param name="lambda_l2" value="0.07"/>
+            <param name="min_gain_to_split" value="0.02"/>
+            <param name="min_child_weight" value="39.0"/>
+            <param name="verbose" value="-1"/>
+            <param name="random_state" value="1"/>
+            <output name="outfile_fit" file="lgb_class_model.txt" compare="sim_size" delta="5"/>
+        </test>
+
+    </tests>
+    <help><![CDATA[
+**What it does**
+
+LightGBM is a gradient boosting framework that uses tree based learning algorithms. It 
+is designed to be distributed and efficient with the following advantages:
+
+* Faster training speed and higher efficiency.
+* Lower memory usage.
+* Better accuracy.
```

Comment:
`Faster, better ...` than which algorithm? If you know, please mention it

Reply:
You are right, this isn't very objective. I have removed this section. Thanks for the comments!